src/PrizeVaultFactory.sol

```diff
@@ -52,42 +68,25 @@ contract PrizeVaultFactory {
     /// yield accrual rates.
     ///
-    /// The yield buffer of vaults deployed by this factory is 1e5. This means that if you deploy a 
-    /// vault with USDC as the underlying asset, you will have to approve this factory to spend 1e5
-    /// USDC ($0.10) to be sent to the prize vault during deployment. This value will cover the first
-    /// 100k rounding errors on deposits and withdraws to the vault and is not recoverable by the 
-    /// deployer.
+    /// This factory will transfer an amount of assets equal to the yield buffer from the deployer to the
+    /// prize vault on deployment to cover the initial buffer. For example, if you are deploying a USDC
+    /// vault and the yield buffer is set to 1e5, you will have to approve this factory to spend 1e5
+    /// USDC ($0.10) to be sent to the prize vault during deployment. Assuming there is no additional 
+    /// precision loss in the yield vault, a 1e5 yield buffer will cover the first 100k rounding errors on
+    /// deposits and withdraws and is not recoverable by the deployer.
     ///
     /// If the yield buffer is depleted on a vault, the vault will prevent any further 
     /// deposits if it would result in a rounding error and any rounding errors incurred by withdrawals
     /// will not be covered by yield. The yield buffer will be replenished automatically as yield accrues
     /// on deposits.
-    uint256 public constant YIELD_BUFFER = 1e5;
-
-    /// @notice List of all vaults deployed by this factory.
-    PrizeVault[] public allVaults;
-
-    /// @notice Mapping to verify if a Vault has been deployed via this factory.
-    mapping(address vault => bool deployedByFactory) public deployedVaults;
-
-    /// @notice Mapping to store deployer nonces for CREATE2
-    mapping(address deployer => uint256 nonce) public deployerNonces;
-
-    ////////////////////////////////////////////////////////////////////////////////
-    // External Functions
-    ////////////////////////////////////////////////////////////////////////////////
-
-    /// @notice Deploy a new vault
-    /// @dev Emits a `NewPrizeVault` event with the vault details.
-    /// @dev The caller MUST approve this factory to spend underlying assets equal to `YIELD_BUFFER` so the yield
-    /// buffer can be filled on deployment. This value is unrecoverable and is expected to be insignificant.
     /// @param _name Name of the ERC20 share minted by the vault
     /// @param _symbol Symbol of the ERC20 share minted by the vault
     /// @param _yieldVault Address of the ERC4626 vault in which assets are deposited to generate yield
     /// @param _prizePool Address of the PrizePool that computes prizes
     /// @param _claimer Address of the claimer
     /// @param _yieldFeeRecipient Address of the yield fee recipient
     /// @param _yieldFeePercentage Yield fee percentage
+    /// @param _yieldBuffer The size of the prize vault yield buffer
```

Comment:
Maybe add the suggested amount here?  `The size of the prize vault yield buffer.  For 18 decimals tokens a safe amount would be 1e5`

Reply:
Suggestions may vary based on the situation, so we can add this info to the factory app so it can be updated with new discoveries.